From 4693a672aa9536774e6d90352f9b4a3d07301fb4 Mon Sep 17 00:00:00 2001
From: "Ranjan, Rajani" <rajani.ranjan@intel.com>
Date: Fri, 11 Aug 2023 21:06:46 +0530
Subject: [PATCH] Remove android.system.wifi.keystore from device
 compatibility_matrix

android.system.wifi.keystore is deprecated as per commit:6aa5899aaa
in system/libhidl with FCM 8. This functionality is replaced by new
methods added to android.hardware.wifi.supplicant.

Tracked-On: OAM-110636
Signed-off-by: Ranjan, Rajani <rajani.ranjan@intel.com>
Signed-off-by: Vilas R K <vilas.r.k@intel.com>
---
 groups/device-specific/caas/compatibility_matrix.xml | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/groups/device-specific/caas/compatibility_matrix.xml b/groups/device-specific/caas/compatibility_matrix.xml
index d9af65e..7f2dab0 100644
--- a/groups/device-specific/caas/compatibility_matrix.xml
+++ b/groups/device-specific/caas/compatibility_matrix.xml
@@ -39,14 +39,6 @@
             <instance>default</instance>
         </interface>
     </hal>
-    <hal format="hidl" optional="false">
-        <name>android.system.wifi.keystore</name>
-        <version>1.0</version>
-        <interface>
-            <name>IKeystore</name>
-            <instance>default</instance>
-        </interface>
-    </hal>
     <!--vendor-ndk>
         <version>33</version>
     </vendor-ndk-->
-- 
2.41.0

